depthmapX/tableView.cpp

```diff
@@ -104,21 +106,23 @@ QSize tableView::sizeHint() const
 	return QSize(2000, 2000);
 }
 
-void tableView::PrepareCache(int from, int to)
+void tableView::PrepareCache(int to)
 {
 	in_update = 1;
 	QTableWidgetItem *Item;
 	const AttributeTable& table = pDoc->m_meta_graph->getAttributeTable();
 	AttributeIndex& index = table.m_display_index;
 
-	if(to+PG_COUNT >= m_row_count) return;
-	for (int i = 0; i < PG_COUNT; i++)
+    int diff = PG_COUNT;
+    if(to+PG_COUNT >= m_row_count) diff = m_row_count - to;

```

Comment:
Please format if statements in separate rows with braces. Always. It takes a bit more space but increases the readability of the code a lot. I know the old code doesn't always do it, but that's just bad practice.